src/main/java/redis/clients/jedis/commands/AdvancedJedisCommands.java

```diff
@@ -64,5 +65,11 @@
 
   List<String> aclCat(String category);
 
+  List<Map<String,String>> aclLog();
```

Comment:
It should be `List<Map<String,Object>>`.

Reply:
@gkorland Should we go for a new custom object here?

Reply:
I have created a custom object for the log entry see `AccessControlLogEntry` class